include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl_radix_sort_one_wg.h

```diff
@@ -111,15 +111,16 @@ struct __subgroup_radix_sort
     {
         assert(__n <= 1 << 16); //the kernel is designed for data size <= 64K
 
+        const auto __req_slm_size_counters = __counter_buf_sz * sizeof(uint32_t);
+
         // Pessimistically only use half of the memory to take into account memory used by compiled kernel
```

Comment:
While we are here, can we make this change to be more specific?

```suggestion
        // Pessimistically only use half of the memory to take into account memory used by group scan algorithm
```

Reply:
I am confused - why "scan algorithm"?

Reply:
I believe the reason we are using only half the SLM is because of the use of `__exclusive_scan_over_group`.  Is it correct, or from some other reason?

This is not necessarily relevant to this PR, but the comment confused me as I was investigating it.

Reply:
Unfortunately, we could not understand what is true reason... and just reduced SLM size in 2 times...
Regarding to "group scan" - I am not sure that is a reason, because "group_scan" is launched here on the private memory, not SLM.

Reply:
I believe `__exclusive_scan_over_group` uses some amount of SLM temporary data, and there is not a way to query how much.  In other places of oneDPL, we have reduced the amount of SLM by half to account for this for both the `...over_group` and `joint_...` with `scan` and `reduce`.  @adamfidel @julianmi  perhaps you can confirm this or provide some more information here?

Reply:
In that case a more clear comment should be look like that:
`// Pessimistically only use half of the memory to take into account a SYCL group algorithm might use a portion of SLM`

Reply:
Yes this is better than my suggestion.